components/builder-web/app/origin/origin-page/_origin-page.component.scss

```diff
@@ -69,7 +65,7 @@
           }
         }
 
-        &:hover {
+        &:not(.heading):hover {
           background-color: rgba($medium-gray, 0.05);
 
```

Comment:
Currently, there is a hover state being applied to the heading of our table-style lists. This change limits the background color/hover effect to list items (rows) other than the heading.